packages/nimble-components/src/table-column/enum-base/models/mapping-spinner-config.ts

```diff
@@ -1,10 +1,36 @@
+import { type ViewTemplate, html } from '@microsoft/fast-element';
 import { MappingConfig } from './mapping-config';
+import type { IconView } from './mapping-icon-config';
+import { spinnerTag } from '../../../spinner';
+
+export interface SpinnerView {
+    text?: string;
+}
+
+const createSpinnerTemplate = (textHidden: boolean): ViewTemplate<IconView> => {
+    return html`
+        <${spinnerTag}
+            title="${x => (textHidden ? x.text : '')}"
+            aria-label="${x => x.text}"
+            aria-hidden="${_ => (textHidden ? 'false' : 'true')}"
+        >
+        </${spinnerTag}>
+    `;
```

Comment:
The goal should be creating as few of these as possible and instead relying on binding updates when we can. Is it possible to have textHidden be part of the IconView/SpinnerView state and place it on the config?

Then this template can be static instead of newly constructed per config, i.e.:

```
html`
        <${spinnerTag}
            title="${x => (x.textHidden ? x.text : '')}"
            aria-label="${x => x.text}"
            aria-hidden="${_ => (xtextHidden ? 'false' : 'true')}"
        >
        </${spinnerTag}>
    `;
```

Reply:
Looks like textHidden is already on the cell views so shouldn't be too bad?